src/main/java/org/apache/geronimo/microprofile/impl/jwtauth/jaxrs/GeronimoJwtAuthExceptionMapper.java

```diff
@@ -36,7 +34,6 @@
     public Response toResponse(final JwtException e) {
         return Response.status(e.getStatus())
                 .entity(builder.toObject(e))
-                .type(APPLICATION_JSON_TYPE)
```

Comment:
Guess we dont want to change that to ensure we serve a type our model is compatible with (only json today)

Reply:
I can live without it. But the response is wrong when using plain text like in the TCK tests. I guess you are simply not getting an error in that mode.

Reply:
k, I removed this change since I can just make sure not the get any errors ;)